common/winobjc.packagereference.override.targets

```diff
@@ -67,9 +78,24 @@
     </Task>
   </UsingTask>
 
-  <Target Name="FixupWinObjCProjectReferences" BeforeTargets="_GetRestoreProjectStyle">
-      <FixupVersionOfWinObjCProjectReferences PackageReferences="@(PackageReference)"
-         SearchPaths="@(WinObjCPackageOverridePath)">
+  <PropertyGroup>
+    <FixupWinObjCProjectReferencesDependsOn>_GetRestoreProjectStyle</FixupWinObjCProjectReferencesDependsOn>
+    <FixupWinObjCProjectReferencesDependsOn Condition="'$(IsNuGetized)' != ''">GetPackageContents;%(FixupWinObjCProjectReferencesDependsOn)</FixupWinObjCProjectReferencesDependsOn>
```

Comment:
yeah that is a little weird
but its so that if we do a dynamic version replacement
we correctly describe what version was used when making a package
this is seen in Frameworks -> Language
right now Frameworks .nuspec says it dpends on "*" for Language even if it was built with a specific version